var/spack/repos/builtin/packages/whizard/package.py

```diff
@@ -20,6 +20,7 @@ class Whizard(AutotoolsPackage):
     maintainers = ['vvolkl']
 
     version('master', branch="master")
+    version('3.0.1', sha256='1463abd6c50ffe72029abc6f5a7d28ec63013852bfe5914cb464b58202c1437c')
     version('3.0.0_alpha', sha256='4636e5a10350bb67ccc98cd105bc891ea04f3393c2420f81be3d21240be20009')
     version('2.8.5', sha256='0f633e5620aa7dd50336b492e8a76bfae15b15943ea842010346ad7610818ecd', preferred=True)
```

Comment:
I assume `preferred=True` was added to prevent the alpha release from being installed by default? Should we remove it so that 3.0.1 is now installed by default?

Reply:
Yes, or most people, there is no reason not to use the `3.x` releases .We'll still use `2.8.5` for quite a while, but I can use the `packages.yaml` for that.